packages/shared/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@redhat-cloud-services/javascript-clients-shared",
+  "version": "1.0.0",
+  "description": "",
+  "main": "./dist/index.js",
```

Comment:
The build command is also building the `esm` version. You should add the module attribute:
```
"module": "./dist/esm/index.js"
```